src/app/legacy/containers/PodcastExternalLinks/index.jsx

```diff
@@ -122,44 +143,42 @@ const PodcastExternalLinks = ({ brandTitle, links }) => {
       </ThirdPartyLinksTitle>
       {hasMultipleLinks ? (
         <StyledList role="list">
-          {links.map(({ linkText, linkUrl }) => (
+          {links.map(({ linkText, linkUrl, linkType }) => (
             <StyledListItem dir={dir} key={`${linkText}`}>
-              <Link
-                // line 126 and id={`externalLinkId-${linkText}`} in line 133 are a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
-                aria-labelledby={`externalLinkId-${linkText}`}
-                href={linkUrl}
-                service={service}
-                script={script}
-                dir={dir}
-                onClick={clickTrackerRef}
+              <PodcastExternalLink
+                linkText={linkText}
+                linkUrl={linkUrl}
+                linkType={linkType}
+                brandTitle={brandTitle}
+                aria={{ 'aria-labelledby': `externalLinkId-${linkText}` }}
               >
                 <span role="text" id={`externalLinkId-${linkText}`}>
                   <span lang={ExternalLinkTextLangs[linkText] || lang}>
                     {linkText}
                   </span>
                   <VisuallyHiddenText>{`, ${brandTitle}${externalLinkText}`}</VisuallyHiddenText>
                 </span>
-              </Link>
+              </PodcastExternalLink>
             </StyledListItem>
           ))}
         </StyledList>
       ) : (
-        <Link
-          aria-label={`${firstLink.linkText}, ${brandTitle} ${externalLinkText}`}
-          href={firstLink.linkUrl}
-          key={firstLink.linkText}
-          service={service}
-          script={script}
-          dir={dir}
-          onClick={clickTrackerRef}
+        <PodcastExternalLink
+          linkText={firstLink.linkText}
+          linkUrl={firstLink.linkUrl}
+          linkType={firstLink.linkType}
+          brandTitle={brandTitle}
```

Comment:
Can remove this prop as it isn't being used by the `PodcastExternalLink` component anymore.

Reply:
fixed in this PR: https://github.com/bbc/simorgh/pull/10544